onnxruntime/python/tools/quantization/operators/softmax.py

```diff
@@ -80,6 +80,11 @@ def quantize(self):
         if self.quantizer.activation_qType == onnx.onnx_pb.TensorProto.UINT8:
             out_scale = 1 / 256.0
             out_zero_point = 0
+        elif self.quantizer.is_activation_symmetric:
```

Comment:
Do we need to support symmetric in QLinearSoftmax also?

Reply:
no, qlinearXXX is the result of Q node merging with others